app/daemon.js

```diff
@@ -126,7 +151,7 @@ export function ensuresIPFSInitialised () {
   if (isIPFSInitialised()) return Promise.resolve()
   console.log('Initialising IPFS repository...')
   return new Promise((resolve, reject) => {
-    const ipfsProcess = spawnIPFSCommand('init')
+    const ipfsProcess = spawnIPFSCommand('init', `--api=${global.IPFS_MULTIADDR_API}`)
```

Comment:
Why do we pass `--api=${global.IPFS_MULTIADDR_API}` here and when starting the daemon? since we call `ensureDaemonConfigured` beforehand, shouldn't it be set?